website/singlepages/locale/nl/LC_MESSAGES/django.po

```diff
@@ -86,10 +90,1205 @@ msgstr ""
+msgid "Version 1.1, 17-04-2019"
+msgstr "Versie 1.1, 17-04-2019"
+
+#: templates/singlepages/event_registration_terms.html
+msgid ""
+"These terms and conditions were drafted to ensure that Thalia doesn’t incur "
+"costs for people who register for an event, but don’t show up. Through this, "
+"there will be more spots at events for people who will actually participate."
+msgstr ""
+"Deze voorwaarden zijn opgesteld om te zorgen dat Thalia geen kosten maakt "
+"voor mensen die zich aanmelden, maar niet komen opdagen bij evenementen en "
+"om zo te zorgen dat er bij deze evenementen meer ruimte is voor leden die "
+"wel komen opdagen."
+
+#: templates/singlepages/event_registration_terms.html
+msgid "1. Stipulations"
+msgstr "1. Bepalingen"
+
+#: templates/singlepages/event_registration_terms.html
+msgid ""
+"A cancellation deadline will be disclosed for each event where a "
+"registration is needed. When a person who is registered wants to cancel "
+"their registration, they should do this prior to the cancellation deadline."
+msgstr ""
+"Bij elk evenement waar een inschrijving benodigd is wordt ook een "
+"afmelddeadline gepubliceerd. Wanneer een ingeschrevene zich wenst af te "
+"melden voor een evenement, dient dit voor de afmelddeadline te gebeuren."
+
+#: templates/singlepages/event_registration_terms.html
+msgid ""
+"When someone doesn’t cancel his registration on time and doesn’t attend the "
+"event, they owe Thalia a compensation of the expenses for the event per "
+"person, with a minimum of €5."
+msgstr ""
+"Wanneer iemand zich niet of na de afmelddeadline afmeldt, maar niet naar het "
+"evenement gaat, is deze persoon Thalia een vergoeding van de gemaakte kosten "
+"per persoon verschuldigd, met een minimum van €5."
+
+#: templates/singlepages/event_registration_terms.html
+msgid ""
+"It is the responsibility of the registered person to make sure that this "
+"person is on the attendance list: it is leading for determining whether they "
+"attended the event."
+msgstr ""
+"Het is de verantwoordelijkheid van de ingeschreven persoon om te zorgen dat "
+"deze persoon op de aanwezigheidslijst staat. De aanwezigheidslijst is "
+"leidend voor het bepalen van de aanwezigheid van een persoon bij een "
+"evenement."
+
+#: templates/singlepages/event_registration_terms.html
+msgid ""
+"The compensation stated on the event page is only an estimate of the actual "
+"figure."
+msgstr ""
+"De vergoeding die op evenementpagina vermeld staat is slechts een schatting "
+"van de uiteindelijke boete."
+
+#: templates/singlepages/event_registration_terms.html
+msgid ""
+"This compensation needs to be paid within two weeks from the invoice date. "
+"If the payment isn’t fulfilled within two weeks from the invoice date, the "
+"person concerned will not be allowed to attend events with the exception of "
+"borrels of S.A. Thalia. If the payment isn’t fulfilled within four weeks "
+"from the invoice date, the person concerned will not be allowed to attend "
```

Comment:
```suggestion
"from the invoice date, the person concerned will also not be allowed to attend "
```

Reply:
This is out-of-scope + changing the translation will make it invalid for the page it is a part of.